src/activities/settings/SettingsActivity.cpp

```diff
@@ -174,9 +177,14 @@ void SettingsActivity::render() const {
   }
 
   // Draw help text
-  renderer.drawButtonHints(UI_FONT_ID, "« Save", "Toggle", "", "");
-  renderer.drawText(SMALL_FONT_ID, pageWidth - 20 - renderer.getTextWidth(SMALL_FONT_ID, CROSSPOINT_VERSION),
-                    pageHeight - 30, CROSSPOINT_VERSION);
+  const auto labels = mappedInput.mapLabels("« Save", "Toggle", "", "");
+  renderer.drawButtonHints(UI_FONT_ID, labels.btn1, labels.btn2, labels.btn3, labels.btn4);
+  if (SETTINGS.swapFrontButtons) {
+    renderer.drawText(SMALL_FONT_ID, 20, pageHeight - 30, CROSSPOINT_VERSION);
+  } else {
+    renderer.drawText(SMALL_FONT_ID, pageWidth - 20 - renderer.getTextWidth(SMALL_FONT_ID, CROSSPOINT_VERSION),
+                      pageHeight - 30, CROSSPOINT_VERSION);
+  }
```

Comment:
Lets just move this text above the button hints to avoid having to move it in future complex cases